### PR TITLE
SAK-29923 Adding macro to repeat post,preview,cancel buttons on top and bottom .

### PR DESCRIPTION
**Pull request by surya-janani**

### CODE REVIEW

**Comment by buckett** [comment]
This doesn't cleanly merge.

**Comment by surya-janani** [comment]
I think I've resolved the conflicts now

File: assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_new_edit_assignment.vm [hunk 1]
```diff
@@ -1,10 +1,25 @@
 <!-- $Id$ -->
 <!-- start: chef_assignments_instructor_new_edit_assignment.vm  -->
+##Defining macro to repeat buttons at the top and bottom of the page
+#macro(buttonBar $submitnotif)
+	<div class="act">
+		<input accesskey ="s" type="button" class="active" name="post" value="$tlang.getString("gen.pos")"
```
Comment (buckett) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_new_edit_assignment.vm]: Looks like you heave leading spaces on your accesskey attributes.

Comment (surya-janani) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_new_edit_assignment.vm]: do you mean the space between : acceskey<space>="s"

File: assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_new_edit_assignment.vm [hunk 2]
```diff
@@ -30,7 +45,7 @@ $(document).ready(function() {
 	 	});
  	#end
 	SetMaxPoint(document.getElementById('$name_GradeType'), document.getElementById('$name_GradePoints'));
-});
+});N
```
Comment (buckett) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_new_edit_assignment.vm]: What's with the N?

**Comment by surya-janani** [comment]
Review changes and rebasing

**Pull request merged**